changelog/seqera-cloud/v25.1.0_cycle11.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Seqera Cloud v25.1.0_cycle10 - cycle12
+date: 2025-05-06
+tags: [seqera cloud]
+---
+
+This changelog includes releases
```

Comment:
I think we can move this list to the end of the release notes as it's a bit long